kayak_core/src/widget_manager.rs

```diff
@@ -384,6 +358,11 @@ impl WidgetManager {
             }
             return self.get_valid_parent(*parent_id);
         }
+        // assert!(node_id.into_raw_parts().0 == 0);
```

Comment:
It works fine until you open and close things. I think because of the If widget which is removing the widget from the tree it creates some issues here.

Reply:
Oh interesting 🤔